dataplug-gcalendar/app/com/hubofallthings/dataplugCalendar/Module.scala

```diff
@@ -11,6 +11,7 @@ package com.hubofallthings.dataplugCalendar
 import akka.actor.{ ActorSystem, Scheduler }
 import com.google.inject.{ AbstractModule, Provides }
 import com.hubofallthings.dataplug.actors.DataPlugManagerActor
+import com.hubofallthings.dataplug.apiInterfaces.authProviders.HatOAuth2Provider
```

Comment:
Should `Hat` be `HAT`?

Reply:
That's a great question actually. I do wonder that when I have to write an acronym, like HAT or JSON. is it Hat or Json? What's your opinion on that?

Reply:
HAT == Hub of All Things, no?

Reply:
Yes but in the contect of using camelCase I don't know what's better as in cleaner/easier to read. `HATOAuth` or `HatOAuth`?

Reply:
I'm fine either way.

Reply:
What do you personally use? In the core plug we have `HatLoginController` for example. Just for consistency I named it HatOAuth. But my OCD has a fatal crash there. Because HAT is the correct way to type HAT but HATOAuth reminds me of NATO or a Japanese character from anime

Reply:
`HatOAuth` wins for me. :)

Reply:
I vote for `HatOAuth` - much lighter mental load to parse the name.